demo/config/targets/SYSTEM/cmd_tlm/meta_cmd_tlm.txt

```diff
@@ -0,0 +1,11 @@
+TELEMETRY SYSTEM META BIG_ENDIAN "System Meta Data Telemetry Packet"
+  ID_ITEM PKTID 0 8 UINT 1 "Packet Id"
+  ITEM CONFIG 16 256 STRING "Configuration Name"
+  APPEND_ITEM OPERATOR_NAME 512 STRING "Operator Name"
+    WRITE_CONVERSION new_packet_log_conversion.rb
+
+COMMAND SYSTEM META BIG_ENDIAN "System Meta Data Command Packet"
```

Comment:
You have to define both, so that In a command packet log it is a command, and in a telemetry packet log it is a telemetry packet.